src/components/download-print-menu/download-print-menu.tsx

```diff
@@ -0,0 +1,171 @@
+import { h, Component } from "preact";
+import { Popover } from "@playkit-js-contrib/ui";
+import { PopoverMenu, PopoverMenuItem } from "../popover-menu";
+import * as styles from "./download-print-menu.scss";
+
+interface DownloadPrintMenuProps {
+    dropdownAriaLabel: string;
+    printButtonAriaLabel: string;
+    downloadButtonAriaLabel: string;
+    onDownload: () => void;
+    onPrint: () => void;
+    downloadDisabled: boolean;
+    printDisabled: boolean;
+}
+
+interface DownloadPrintMenuState {
+    popoverOpen: boolean;
+}
+
+export class DownloadPrintMenu extends Component<DownloadPrintMenuProps, DownloadPrintMenuState> {
+    static defaultProps = {
+        dropdownAriaLabel: "Download or print transcript",
```

Comment:
@semarche-kaltura 
Don't we need the phrase "current" here too?

Reply:
done